Gruntfile.js

```diff
@@ -136,11 +138,38 @@ module.exports = (grunt) => {
     }
   }
 
+  async function redeployAndUpdateMainSrc (manifestDir) {
+    if (development) {
+      const { API_URL, GI_PERSIST } = process.env
+      const { dirname, filename } = databaseOptionBags[GI_PERSIST] ?? {}
+      let dest = './data'
+      if (GI_PERSIST === 'sqlite') dest = path.join(dirname, filename)
+      if (GI_PERSIST === 'fs') dest = dirname
+      if (!GI_PERSIST) dest = API_URL
```

Comment:
AFAICT, this code can be simplified significantly.

1. We can start by setting `let dest = API_URL`. This eliminates the need for line 148
2. Next, since the variables `dirname` and `filename` are never used for anything except to set `dest`, we can further simplify by directly retrieving the full path from `databaseOptionBags` and having it there. This eliminates the need for lines 146 and 147